### ISSUE TITLE
key error when using run_classifier.py in predict mode, expecting label?

### ISSUE THREAD

**Issue by search4mahesh** [huggingface/pytorch-pretrained-BERT#576]
Hi,

I am getting key error when using run_classifier.py in predict mode.

https://github.com/huggingface/pytorch-pretrained-BERT/blob/master/examples/run_classifier.py

At prediction time we don't have labels hence it gives key error.
run_squad example is good as it was having is_training flag.
Could you please suggest?

if output_mode == "classification":
            label_id = label_map[example.label]
        elif output_mode == "regression":
            label_id = float(example.label)
        else:
            raise KeyError(output_mode)

Thanks
Mahesh
